File: Ruby/bubble_sort.rb
def bubble_sort(arr)
  i = 0
  while i < arr.length
    j = i + 1
    while j < arr.length
      arr[i], arr[j] = arr[j], arr[i] if arr[i] > arr[j]

      j += 1
    end

    i += 1
  end

 arr
end

File: Javascript/bubble_sort.js
const bubble_sort = function(arr) {
  for(let i = 0; i < arr.length; i++){
    for(let j = 0; j < arr.length; j++){
        let temp = arr[i];

        if(arr[i] < arr[j]){
          arr[i] = arr[j];
          arr[j] = temp;
        }
    }
  }

  return arr
}

array = [5,4,4,3,1,24,5,67,9,0]
bubble_sort(array);
